tests/client/amo/components/TestAddonDetail.js

```diff
@@ -6,8 +6,9 @@ import {
 } from 'react-addons-test-utils';
 import { Provider } from 'react-redux';
 
-import AddonDetail, { allowedDescriptionTags }
-  from 'amo/components/AddonDetail';
+import AddonDetail, {
+  allowedDescriptionTags,
+} from 'amo/components/AddonDetail';
```

Comment:
Looks like this could fit on one line. It's also probably why you have a conflict so maybe just leave it as-is.